.gitignore

```diff
@@ -65,4 +65,5 @@ _SpecRunner.html
 .DS_Store
 
 # Ignore xlsx files generated during testing
-*.xlsx
\ No newline at end of file
+*.xlsx
+*{}
```

Comment:
what this (`*{}`) mean?

Reply:
These .gitignore items I added for test artifacts.